cmd/locode/db/csvlocodedb/calls.go

```diff
@@ -0,0 +1,158 @@
+package csvlocodedb
+
+import (
+	_ "embed"
+	"encoding/csv"
+	"errors"
+	"os"
+	"sort"
+	"strconv"
+
+	locodedb "github.com/nspcc-dev/locode-db/cmd/locode/db"
+)
+
+var errRecordNotFound = errors.New("record not found")
+
+// Get returns a record by key. locodedb.Key is a struct that contains CountryCode and LocationCode.
+// locodedb.Record is a struct that contains CountryName, LocationName, SubDivName, SubDivCode, GeoPoint and Continent.
+func (db *DB) Get(key locodedb.Key) (rec *locodedb.Record, err error) {
```

Comment:
Getter should be a separate package with an interface like
```
func Get(locode string) (rec *Record, err error)
```
I'd expect most users to have `string` first, not `Key`, but this can be adjusted (or two functions can be provided).